### PR TITLE
CY-3518 - Backport PM2 drop to 5.0.5

### PR DESCRIPTION
**Pull request by qooban**

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -104,17 +103,15 @@ app.use(
...
     `${contextPath}/userData`,
     passport.authenticate('cookie', { session: false }),
-    expressStaticGzip(path.resolve(__dirname, '../dist/userData'), { enableBrotli: true, indexFromEmptyFile: false })
+    expressStaticGzip(
+        path.resolve(__dirname, process.env.NODE_ENV === 'development' ? '../userData' : '../dist/userData'),
+        {
+            enableBrotli: true,
+            indexFromEmptyFile: false
+        }
+    )
```
Comment (kubama) [inline backend/server.js:112]: How is this change related to pm2 drop?

Comment (qooban) [reply backend/server.js]: Good catch. Added by mistake during cherry-picking. Fixed.

Comment (kubama) [reply backend/server.js]: I guess this should be addressed in https://github.com/cloudify-cosmo/patchify/pull/193 as well, right?

**Review by kubama — APPROVED** [review]

File: backend/server.js (hunk 1 above)
Comment (qooban) [reply backend/server.js]: Yes. Working on it.
